### PR TITLE
New Line to index.html

### PR DESCRIPTION
**Pull request by Eeriena**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,7 +13,7 @@ <h1>Hello world!</h1>
 <p>A new line in our file!</p>
 <p>Another Line!</p>
 <p>This line is here to show how merging works.</p>
-
+<h2>ATTN!NEW Information</h2>
```
Comment (gusevda) [inline index.html]: Let's add space after ATTN!

**Review by gusevda — CHANGES_REQUESTED** [review]
One moment to check

File: index.html (hunk 1 above)
Comment (Eeriena) [reply index.html]: I don't think so!

**Review by gusevda — APPROVED** [review]
Looks great!

**Review by gusevda — APPROVED** [review]

**Pull request merged**